packages/core/tracing/src/components/waterfall/WaterfallSpanRow.vue

```diff
@@ -0,0 +1,305 @@
+    config.selectedSpan = undefined
+  } else {
+    config.selectedSpan = props.spanNode
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.waterfall-span-row {
+  box-sizing: border-box;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  &.selected {
+    $row-background-color: $kui-color-background-primary-weakest;
+    background-color: $row-background-color;
+
+    .bar-wrapper {
+      &::before {
+        background: linear-gradient(to right,
+        $row-background-color 0%,
+        rgba($row-background-color, 0) 50%,
+        rgba($row-background-color, 0) 100%);
+      }
+
+      &::after {
+        background: linear-gradient(to left,
+        $row-background-color 0%,
+        rgba($row-background-color, 0) 50%,
+        rgba($row-background-color, 0) 100%);
+      }
+    }
+  }
+
+  .label {
+    align-items: center;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    font-size: $kui-font-size-20;
+    justify-content: flex-start;
+    position: relative;
+    width: 100%;
+
+    .label-content {
+      align-items: center;
+      box-sizing: border-box;
+      cursor: pointer;
+      display: flex;
+      flex-direction: row;
+      gap: $kui-space-30;
+      justify-content: space-between;
+      min-width: 0;
+      padding: $kui-space-20 $kui-space-10;
+      position: relative;
+      width: 100%;
+
+      &::after {
+        border-bottom: 1px solid $kui-color-border-neutral-weaker;
+        bottom: 0;
+        content: '';
+        left: 0;
+        position: absolute;
+        width: 100%;
+      }
+
+      .name {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+      }
+
+      .end-decorator {
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        gap: $kui-space-20;
+        justify-content: flex-end;
+
+        .duration {
+          font-size: $kui-font-size-20;
+        }
+      }
+    }
+  }
+
+  &:last-child {
+    .label-content {
+      &::after {
+        border-bottom: none;
+      }
+    }
+  }
+
+  .bar-wrapper {
+    align-items: center;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    font-size: $kui-font-size-30;
+    justify-content: flex-start;
+    overflow: hidden;
+    padding: $kui-space-20 v-bind(WATERFALL_SPAN_BAR_FADING_WIDTH);
+    position: relative;
+
+    &::before {
+      background: linear-gradient(to right,
+      white 0%,
+      rgba(255, 255, 255, 0) 50%,
+      rgba(255, 255, 255, 0) 100%);
+      content: "";
+      display: block;
+      height: 100%;
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: v-bind(WATERFALL_SPAN_BAR_FADING_WIDTH);
+      z-index: 10;
+    }
+
+    &::after {
+      background: linear-gradient(to left,
+      white 0%,
+      rgba(255, 255, 255, 0) 50%,
+      rgba(255, 255, 255, 0) 100%);
+      content: "";
```

Comment:
Are we preferring single quotes or double quotes in CSS? I saw both usage in this PR and across the codebase 😅

Reply:
It seems neither ESLint (or Stylelint) nor "Vue - Official" have a preference for this. I'm updating them with single quotes as we seem to prefer them in JS/TS codes.